dist/dropzone-amd-module.js

```diff
@@ -673,6 +679,12 @@
                 if ((clickableElement !== _this.element) || (evt.target === _this.element || Dropzone.elementInside(evt.target, _this.element.querySelector(".dz-message")))) {
                   return _this.hiddenFileInput.click();
                 }
+              },
+              "touchstart": function(evt) {
+                evt.stopPropagation();
```

Comment:
Would it make sense to use `noPropagation(evt)` here? It's being [used elsewhere](https://github.com/rodneyrd/dropzone/blob/master/dist/dropzone-amd-module.js#L620-L627).